modules/media/jsx/editForm.js

```diff
@@ -140,7 +142,7 @@ class MediaEditForm extends Component {
           />
           <SelectElement
             name='instrument'
-            label='Instrument'
+            label={t('Instrument', {ns: 'media'})}
```

Comment:
Instrument is in the loris namespace (but needs a `count:1` because it has a plural form)